platform/src/main/java/org/stellar/anchor/platform/config/PropertyClientsConfig.java

```diff
@@ -84,9 +104,11 @@ private void validateClient(ClientConfig clientConfig, Errors errors) {
   }
 
   public void validateCustodialClient(ClientConfig clientConfig, Errors errors) {
-    if (isEmpty(clientConfig.getSigningKey())) {
+    if (isEmpty(clientConfig.getSigningKey())
```

Comment:
Should we keep the validation until we remove the deprecated fields?

Reply:
It's intended to remove `SigningKey` for non-custodial client. Only custodial wallets will need the public_key attribute per discussion.